### DISCUSSION TITLE
jupyter lab can't open

### DISCUSSION THREAD

**Discussion by fandsyah99** [giswqs/geog-312#27] [Q&A] [answered]
my jupyter can't open it say " 'jupyter' is not recognized as an internal or external command,
operable program or batch file. " how to fix that?

**Comment by giswqs** [comment, marked as answer]
pip install jupyterlab

**Reply by fandsyah99** [reply]
thank you sir

**Comment by Geozike** [comment]
Sir, i have similar problem. Kindly look at mine after installing the jupyter lab. 
![problem 4](https://github.com/user-attachments/assets/d8033354-1ab5-489f-a94a-6abc8bee3339)
